src/SqlAsyncCollector.cs

```diff
@@ -171,6 +172,16 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
                 ObjectCache cachedTables = MemoryCache.Default;
                 var tableInfo = cachedTables[cacheKey] as TableInformation;
 
+                int timeout = 10;
+                string timeoutEnvVar = Environment.GetEnvironmentVariable("AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MIN");
+                this._logger.LogDebugWithThreadId("TIMEOUT STRING HERE " + timeoutEnvVar);
+
+                if (!string.IsNullOrEmpty(timeoutEnvVar))
+                {
+                    timeout = int.Parse(timeoutEnvVar, CultureInfo.InvariantCulture);
```

Comment:
TryParse - we don't want this to fail if the variable isn't set correctly. Just fall back to the default in that case.